.travis.yml

```diff
@@ -8,9 +8,11 @@ install:
 - bundle install
 before_script:
 - bundle exec hatchet ci:setup
-script: bundle exec parallel_rspec -n6 test/spec/
+- heroku update
+script: bundle exec parallel_rspec -n 7 --runtime-log "test/var/log/parallel_runtime_rspec.${STACK}.log" --verbose-process-command --combine-stderr --prefix-output-with-test-env-number test/spec/
```

Comment:
I don't understand this, but will assume you know what you're doing

Reply:
There's four separate commits in this PR that all touch this single line, each with an explanation, if you're curious :)